src/deluge/gui/views/instrument_clip_view.h

```diff
@@ -134,8 +134,25 @@ class InstrumentClipView final : public ClipView, public InstrumentClipMinder {
 	void dontDeleteNotesOnDepress();
 
 	void tempoEncoderAction(int8_t offset, bool encoderButtonPressed, bool shiftButtonPressed);
+	void sendAuditionNote(bool on, uint8_t yDisplay, uint8_t velocity, uint32_t sampleSyncLength);
```

Comment:
Instead of modifying all the call sites, the second 2 parameters can be defaulted (`velocity = 64, uint32_t sampleSyncLength = 0`).

Reply:
Oh I wasn't modifying the call sites. that code was already there in the slicer function. sendAuditionNote was previously a private function and whoever created that slicer code added an inline public function to this header file so that they could call that private function. I removed that inline and made this sendAuditionNote function public as I need to use it in my view as well.